_includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md

```diff
@@ -0,0 +1,1273 @@
+
+Install the required modules using Composer:
+
+```bash
+composer require spryker-feature/customer-account-management: "{{page.version}}" --update-with-dependencies
+```
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules have been installed:
+
+| MODULE                        | EXPECTED DIRECTORY                                   |
+|-------------------------------|------------------------------------------------------|
+| CustomerPage                  | vendor/spryker-shop/customer-page                    |
+| CustomerValidationPage        | vendor/spryker-shop/customer-validation-page         |
+| SessionCustomerValidationPage | vendor/spryker-shop/session-customer-validation-page |
+
+
+{% endinfo_block %}
+
+### 2) Set up configuration
+
+Optional: To enable double opt-in for customer registration, adjust the `CustomerPageConfig::isDoubleOptInEnabled()` method to return `true`.
+
+```php
+<?php
+
+namespace Pyz\Yves\CustomerPage;
+
+use SprykerShop\Yves\CustomerPage\CustomerPageConfig as SprykerCustomerPageConfig;
+
+class CustomerPageConfig extends SprykerCustomerPageConfig
+{
+    /**
+     * @return bool
+     */
+    public function isDoubleOptInEnabled(): bool
+    {
+        return true;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that double opt-in is enabled by registering a new customer and checking that you can't sign in to your account until the registration is confirmed.
+
+{% endinfo_block %}
+
+2. By default, posting the login form (where SecurityBlocker makes its check and blocks users who made too many failed login attempts) is locale-independent. To see error messages translated into different languages, you need to configure the locale to be added to the login path. You can do this by modifying the following configs:
+
+{% info_block warningBox %}
+For security reasons, we recommend enabling the security blocker feature that block recurring attempts of resetting a password by setting `CUSTOMER_SECURITY_BLOCKER_ENABLED` to `true;`.
+{% endinfo_block %}
+
+**src/Pyz/Yves/CustomerPage/CustomerPageConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\CustomerPage;
+
+use Spryker\Yves\CustomerPage\CustomerPageConfig as SprykerCustomerPageConfig;
+
+class CustomerPageConfig extends SprykerCustomerPageConfig
+{
+    /**
+     * @var bool
+     */
+    protected const CUSTOMER_SECURITY_BLOCKER_ENABLED = true;
+
+    /**
+     * @return bool
+     */
+    public function isLocaleInLoginCheckPath(): bool
+    {
+        return true;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that, when a customer submits the login form, the URL it uses contains a locale code. For example, `/de/login_check` is the default value for the customer.
+
+{% endinfo_block %}
+
+3. You can define the password minimum and maximum length the frontend can use. They must use the same as you defined for the core feature in the previous `CustomerConfig`:
```

Comment:
Is this optional?

Reply:
yes, core module provides a default value